docs/docs/components/migrating/README.md

```diff
@@ -0,0 +1,190 @@
+# Migrating from Legacy Actions to Component Actions
+
+- [Overview](#overview)
```

Comment:
Consider `[[toc]]`

Reply:
updated. most of these manually generated toc's were remnants from the github readme files. thanks for the tip.